ethcore/Cargo.toml

```diff
@@ -54,7 +54,7 @@ rand_xorshift = "0.1.1"
 rayon = "1.1"
 rlp = "0.4.0"
 rlp_derive = { path = "../util/rlp-derive" }
-rustc-hex = { version = "1", optional = true }
+rustc-hex = "1"
```

Comment:
Unless there's a better way to do hexing, can we use `v2` here?

Reply:
well, I missed this one. Will fix it but looks like it needs some more type annotations..